packages/components/table/src/table-column/defaults.ts

```diff
@@ -145,4 +146,8 @@ export default {
       )
     },
   },
+  diagonal: {
+    type: Boolean,
+    default: false,
```

Comment:
The default value may be removed, refer to the [link](https://vuejs.org/guide/components/props.html#prop-validation). `The Boolean absent props will be cast to false`.

Reply:
Thank you, the default value have been removed